doc/rule-descriptions.md

```diff
@@ -120,10 +120,6 @@ Rules we are still testing and developing. They are not enabled by default in ax
 
 Deprecated rules are not enabled by default and will be removed in the next major release.
 
-| Rule ID                                                                                                                  | Description                                                                                                                | Impact   | Tags                                                                             | Issue Type        |
-| :----------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------- | :------- | :------------------------------------------------------------------------------- | :---------------- |
-| [aria-dpub-role-fallback](https://dequeuniversity.com/rules/axe/3.5/aria-dpub-role-fallback?application=RuleDescription) | Ensures unsupported DPUB roles are only used on elements with implicit fallback roles                                      | Moderate | cat.aria, wcag2a, wcag131, deprecated                                            | failure           |
-| [checkboxgroup](https://dequeuniversity.com/rules/axe/3.5/checkboxgroup?application=RuleDescription)                     | Ensures related &lt;input type=&quot;checkbox&quot;&gt; elements have a group and that the group designation is consistent | Critical | cat.forms, best-practice, deprecated                                             | failure           |
-| [layout-table](https://dequeuniversity.com/rules/axe/3.5/layout-table?application=RuleDescription)                       | Ensures presentational &lt;table&gt; elements do not use &lt;th&gt;, &lt;caption&gt; elements or the summary attribute     | Serious  | cat.semantics, wcag2a, wcag131, deprecated                                       | failure           |
-| [radiogroup](https://dequeuniversity.com/rules/axe/3.5/radiogroup?application=RuleDescription)                           | Ensures related &lt;input type=&quot;radio&quot;&gt; elements have a group and that the group designation is consistent    | Critical | cat.forms, best-practice, deprecated                                             | failure           |
-| [video-description](https://dequeuniversity.com/rules/axe/3.5/video-description?application=RuleDescription)             | Ensures &lt;video&gt; elements have audio descriptions                                                                     | Critical | cat.text-alternatives, wcag2aa, wcag125, section508, section508.22.b, deprecated | needs&nbsp;review |
+| Rule ID | Description | Impact | Tags | Issue Type |
```

Comment:
This should have better text than just column headers. Something like `None` would be better. Wdyt?